app/services/userValidator.js

```diff
@@ -0,0 +1,30 @@
+const validations = {
```

Comment:
I like this, but i would not call it a service. In wolox we call services to objects that interacts with external services. I think this can be a helper or a validator. If you have another idea, let me know

Reply:
I think a `helper` is fine, but I wasn't sure if I was able to create directories on my own, so I put it in the only directory that made sense. I'll move it then

Reply:
You can :)